src/modules/rlm_imap/all.mk

```diff
@@ -0,0 +1,50 @@
+#######################################################################
```

Comment:
This file is autogenerated by the `configure` script, from the `all.mk.in` template.  It shouldn't be committed to version control.

Reply:
`all.mk`/`all.mk/in` can be removed entirely.  We don't need a configure script here, we can piggyback off the checks done by `libfreeradius-curl`